internal/cfgplugins/policyforwarding.go

```diff
@@ -878,3 +882,68 @@ func ConfigureDutWithGueDecap(t *testing.T, dut *ondatra.DUTDevice, guePort int,
 		// transport.SetDestinationPort()
 	}
 }
+
+// PbrRule defines a policy-based routing rule configuration
+type PbrRule struct {
+	Sequence  uint32
+	EtherType oc.NetworkInstance_PolicyForwarding_Policy_Rule_L2_Ethertype_Union
+	EncapVrf  string
+}
+
+// NewPolicyForwardingVRFSelection creates policy-based routing configuration for VRF selection
+func NewPolicyForwardingVRFSelection(dut *ondatra.DUTDevice, name string) *oc.NetworkInstance_PolicyForwarding {
```

Comment:
All cfgpugins exported/public functions from a cfgplugin should have a parameter signature like:
`(t *testing.T, dut *ondatra.DUTDevice, sb *gnmi.SetBatch, cfg MyConfigToUpdateStruct)`

Reply:
The required style for cfgplugins is the caller is expected to pass a gnmi.Batch object by reference which the cfgplugin function will add additional OC configuration to.   The current form of `NewPolicyForwardingVRFSelection` generates OC configuration and returns it, but that is different than the standard.

I've [updated the cfgplugin README](https://github.com/openconfig/featureprofiles/pull/4667) to make this more explicit (previously, you'd have to infer this from the referenced example code).